convert/src/routers/text.py

```diff
@@ -0,0 +1,30 @@
+from typing import Dict
+
+from fastapi import APIRouter, status, HTTPException
+
+from src.config import minio_client
+from src.logger import get_logger
+from src.vertex_to_bd.models.text_model import TextRequest
+from src.vertex_to_bd.text_to_badgerdoc_use_case import TextToBDConvertUseCase
+
+router = APIRouter(prefix="/text", tags=["text"])
+LOGGER = get_logger(__file__)
+
+
+@router.post(
+    "/import",
+    status_code=status.HTTP_201_CREATED,
+)
+def import_text(request: TextRequest) -> Dict[str, int]:
+    try:
+        text_to_bd_use_case = TextToBDConvertUseCase(
+            s3_client=minio_client,
+        )
+        text_to_bd_use_case.execute(
+            s3_input_text=request.input_text,
+            s3_output_pdf=request.output_pdf,
+            s3_output_tokens=request.output_tokens,
+        )
+    except Exception as exp:
+        raise HTTPException(status_code=500, detail=str(exp))
```

Comment:
May you make exception handling more specific? Python allows you to handle several exception with their own messages with a single exception handler.
And exposing exception messages to the user may cause unintentional leakage of sensitive data

Reply:
fixed